src/registry_schemas/schemas/party.json

```diff
@@ -31,6 +31,20 @@
                     "type": "string",
                     "maxLength": 50
                 },
+                "incorporationNumber": {
+                  "type": "string",
+                  "title": "The Incorporation Number if available for Organization",
+                  "default": "",
+                  "examples": ["CP1234567"],
+                  "pattern": "^[A-Z]{1,3}[0-9]{7}$"
+                },
+                "taxId": {
+                  "type": "string",
+                  "title": "The CRA business number",
+                  "default": "",
+                  "examples": ["123456789"],
+                  "pattern": "^[0-9]{9}$"
+                },
```

Comment:
These are required in party to support multiple organization. example: for General Partnership at least 2 Partners are required and each partner could have these details

Reply:
incorporationNumber is not a schema.org name, it should be _ identifier_
if they are a BC registered business, and you have their _identifier_, we can lookup the taxId as opposed to having it entered here. If it is entered wrong, do we check?